test/Parse/multiline_errors.swift

```diff
@@ -0,0 +1,38 @@
+// RUN: %target-typecheck-verify-swift
+
+import Swift
+
+// ===---------- Multiline --------===
+
+_ = """
+    Eleven
+  Mu
+    """ // expected-error@-1{{invalid mix of multi-line string literal indentation}}
+
```

Comment:
Note for subsequent quality-of-life work (but not strictly needed to approve this PR): please add notes to describe why this is invalid.

E.g.:
`expected-note@-1{expecting at least 4 columns of leading indentation}`

Bonus points: add a fixit to insert the extra columns.